winter-website.html

```diff
@@ -13,22 +13,40 @@
 <body>
 	
 	<div class="navBar">
-		<div class="navBarLinks">
-			<a class="navBarText" href="#">NavBar</a>
-			<a class="navLinkText" href="#">Link</a>
-			<a class="navLinkText" href="#">Link</a>
-			<a class="navLinkText" href="#">Link</a>
-		</div>
+				<a class="navLink" href="#">Home</a>
+				<a class="navLink" href="#">What I Do</a>
+				<a class="navLink" href="#">Things I've Made</a>
+				<a class="navLink" href="#">Contact</a>
+	</div>
+	
+	<div class="sectionAll">
+		<section class="aboutMeSection">
+			<h1>
+				Winter<br />
```

Comment:
br no est beuno hombre

Reply:
force it to have really big margins and it should spread over 2 lines

Reply:
looked into this and i think this is an instance where <br> is acceptable because otherwise it would make the widths of other elements on the website inconsistent